build.js

```diff
@@ -0,0 +1,29 @@
+const rollup = require('rollup');
```

Comment:
I never meant to actually commit this file (or the updated dependencies in package.json). This was there so that you could try to run the config I was.